### PR TITLE
Create UX/Usability section 

### PR DESCRIPTION
**Pull request by Mohamed3on**
* Add Don't Make Me Think by Steve Krug, a great book on software usability, especially on web/mobile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ But knowing the stuff will help you become better! :muscle:*
 
 ### Falsehoods
 - [Awesome Falsehoods](https://github.com/kdeldycke/awesome-falsehood)  
-  💊 Curated list of falsehoods programmers believe in.
```
Comment (mr-mig) [inline README.md]: Please, don't remove the emoji

File: README.md [hunk 2]
```diff
@@ -147,7 +149,7 @@ But knowing the stuff will help you become better! :muscle:*
 ### Papers on Programming
 - :heart: [Papers We Love](https://github.com/papers-we-love/papers-we-love)
 - :newspaper: [The Morning Paper](https://blog.acolyer.org/)
-- 📜 [What Every Programmer Should Know About Memory](http://futuretech.blinkenlights.nl/misc/cpumemory.pdf)
```
Comment (mr-mig) [inline README.md]: Please, don't remove the emoji

**Comment by mr-mig** [comment]
Great idea! 
Could you please clean up the PR?

File: README.md (hunk 2 above)
Comment (Mohamed3on) [reply README.md]: Sorry! I had no idea what did that.

**Comment by Mohamed3on** [comment]
Done!

**Comment by eamanu** [comment]
Great idea!

**Pull request merged**